include/core/serial_solid.h

```diff
@@ -0,0 +1,243 @@
+/* ---------------------------------------------------------------------
+ *
+ * Copyright (C) 2019 - 2019 by the Lethe authors
+ *
+ * This file is part of the Lethe library
+ *
+ * The Lethe library is free software; you can use it, redistribute
+ * it, and/or modify it under the terms of the GNU Lesser General
+ * Public License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * The full text of the license can be found in the file LICENSE at
+ * the top level of the Lethe distribution.
+ *
+ * ---------------------------------------------------------------------
+
+ *
+ * Authors: Carole-Anne Daunais, Valérie Bibeau, Polytechnique Montreal, 2020
+ * Jeanne Joachim, Polytechnique Montreal, 2020-
+ */
+
+#ifndef lethe_serial_solid_base_h
+#define lethe_serial_solid_base_h
+
+
+// Lethe Includes
+#include <core/parameters.h>
+#include <core/pvd_handler.h>
+#include <core/simulation_control.h>
+#include <core/solid_objects_parameters.h>
+
+// Dealii Includes
+#include <dem/data_containers.h>
```

Comment:
You can't include anything from DEM into core. The libraries are made to be exclusive. If you need these data containers here, move them to core.

Reply:
Moved to core.